engine/src/test/java/org/camunda/bpm/engine/test/api/history/HistoryServiceTest.java

```diff
@@ -107,6 +108,78 @@ public void testHistoricProcessInstanceQueryOrderBy() {
     historyService.createHistoricTaskInstanceQuery().orderByTaskPriority().asc().list();
   }
 
+  @Deployment(resources = {"org/camunda/bpm/engine/test/api/oneTaskProcess.bpmn20.xml"})
+  public void testHistoricTaskInstanceQueryTaskNameCaseInsensitive() {
+    // With a clean ProcessEngine, no instances should be available
+    assertTrue(historyService.createHistoricProcessInstanceQuery().count() == 0);
```

Comment:
This assertion is not needed, the database is cleared after each test and there are built-in assertions that will fail if there is some data remaining.
```suggestion
    // given
```